### PR TITLE
OP-706 Change "DICOM" to more general "Imaging"

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Update text and images

### CODE REVIEW

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -1614,22 +1621,24 @@ dicom.max.size=DICOM_SIZE
...
 * *OH_PATH_SUBSTITUTE/DICOM_DIR*: if the value of the first parameter is set to "FileSystemDicomManager",
-this property specifies the path where the DICOM files are stored. It is possible to specify a local folder (_data/dicom_storage_ is the default path under the OH installation folder) or to set a shared network folder, useful for LAN - client/server environment. Suggested configuration is to map the network/NFS folder under the _data/dicom_storage_ path with a symbolic link.
-The shared folder needs read/write permissions for uploading DICOM images. Always use slash "/" in the path variable and not backslash "\".
-* *DICOM_SIZE*: maximum allowed size for a dicom/jpg image (the MariaDB/MySQL server setting max_allowed_packet should be set with the same or a larger value).
+this property specifies the path where the image files are stored. It is possible to specify a local folder (_data/image_storage_ is the default path under the OH installation folder) or to set a shared network folder, useful for LAN - client/server environment. Suggested configuration is to map the network/NFS folder under the _data/image_storage_ path with a symbolic link.
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc:1627]: also this one

**Review by mwithi — COMMENTED** [review]
I think also buttons in the menu are now changed... arent't they?

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (dbmalkovsky) [reply doc_admin/AdminManual.adoc:1627]: I'm confused; I've replaced "DICOM files" with "image files"; "dicom_storage_" with "image_storage_"; what is left to do?

File: doc_admin/AdminManual.adoc [hunk 2]
```diff
@@ -587,7 +587,7 @@ Open Hospital has a folder tree hierarchy which is shown here, where *_<version>
 * *data/conf* – contains the DBMS configuration files
 * *data/db* – contains the database files (PORTABLE mode)
 * *data/logs* – contains all the log files
-* *data/dicom* – contains the dicom files
+* *data/images* – contains the image files
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc:590]: this would also affect the scripts, isn't it @mizzioisf ?

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc:1627]: sorry, there was a pending comment, see above https://github.com/informatici/openhospital-doc/pull/180/files#r1028432221

**Comment by dbmalkovsky** [comment]
Update the image in the User Guide and found a few more references to DICOM.

File: doc_admin/AdminManual.adoc (hunk 2 above)
Comment (dbmalkovsky) [reply doc_admin/AdminManual.adoc:590]: Yes and we've previously discussed that some changes would be needed....

Comment (mwithi) [reply doc_admin/AdminManual.adoc:590]: I was checking the issue for that, but I didn't see trace and also label "scripts" has not been applied yet

**Comment by dbmalkovsky** [comment]
Closing this request to recreate it from scratch.

**Pull request closed without merging**